blackbox/docs/sql/reference/create_ingest_rule.txt

```diff
@@ -0,0 +1,90 @@
+.. _create-ingest-rule:
+
+======================
+``CREATE INGEST RULE``
+======================
+
+Defines a new ingestion rule.
+
+Synopsis
+========
+.. highlight:: psql
+
+::
+
+    CREATE INGEST RULE rule_name
+    ON source_ident
+    [WHERE condition]
+    INTO table_ident
+
+
+Description
+===========
+
+``CREATE INGEST RULE`` creates a new ingestion rule, applying on a
+``source_ident`` and ingesting into the specified target ``table_ident``.
+
+The ingestion rules will be used by ingestion implementations to filter the
+``source_ident`` stream, the filtered data will then be stored in the
+corresponding target ``table_ident``.
+
+Parameters
+==========
+
+:rule_name: The identifier of the rule to create.
+
+:source_ident: The registration identifier of an ingestion source.
+
+:condition: A boolean expression using references specific to the ingestion
+            implementation.
+
+:table_ident: The identifier of the target table.
+
+
+Notes
+=====
+
+.. NOTE::
+
+    When the :ref:`administration_user_management` is enabled, this statement
+    can only be issued by a superuser.
+
+    The default user for the ``INSERT`` operations in the target
+    ``table_ident`` is the superuser ``crate``.
+
+Ingestion rules can be created and dropped, even if no ingestion implementation
+is available. In such case the ingestion rules will not take any effect, until
+the ingestion implementation is registered.
+
+There is no default ingestion rule,  all incoming streams into any ingestion
+endpoint should be discarded by default by any ingestion implementation.
+
+The target table must be created manually, and should contain the same columns
```

Comment:
"same structure" instead of "same columns" ( as the ingestion implementation's data structure is not composed of columns, so "same" does not apply if we use columns)